src/retest/runner.c

```diff
@@ -75,6 +75,11 @@ print(const struct fsm *fsm, enum implementation impl,
 		return 0;
 	}
 
+	/* the vmc codegen can emit memcmp() calls */
+	if (impl == IMPL_VMC && fsm_getoptions(fsm)->io == FSM_IO_PAIR) {
+		fprintf(f, "#include <string.h>\n\n");
```

Comment:
@katef is `\n\n` on purpose here?

Reply:
Yep! it's nice to separate headers from the body of code a bit, just a matter of style